django_fido/js/fido2.js

```diff
@@ -139,14 +139,20 @@ async function sendFido2Request(url, is_registration, credenetials_name, form_da
     }
 }
 
-async function sendFido2RegistrationRequest(url, form_data) {
-    await sendFido2Request(url, true, 'excludeCredentials', form_data)
-}
-
 async function sendFido2AuthenticationRequest(url, form_data) {
     await sendFido2Request(url, false, 'allowCredentials', form_data)
 }
 
+const sendFido2RegistrationRequest = async(e) => {
+    e.preventDefault()
+    const form = document.getElementById(DJANGO_FIDO_FORM_ID)
```

Comment:
I would keep it consistent when it comes to `querySelector` and `getElementById`. I think `querySelector` is overall a better option, as it is more flexible and could be used for anything.